docs/creature_formations.md

```diff
@@ -142,25 +64,8 @@ Sets group member behaviours, values are:
 
 These values are used to set leaderGUID pre ending path points for memberGUID's where the path is a straight return path and memberGUID's should not crossover to other side of leaderGUID on direction change.
 
-If your leader has a path like the one below where he moves to point 5 then back to 1 you would set point\_1 = 4 and point\_2 = 8 on the memberGUID. If the memberGUID is at angle 90 up to point 5 it will switch to angle 270 for the return trip. This is only needed to keep creatures on the correct side. these values can be left at 0 for creatures following directly behind leaderGUID or any creatures in a circular path. 
-
- 
-
-1     2     3     4    5
-
------&lt;---------&gt;------
-
-       8    7      6
-
```

Comment:
Why do you remove this?

Reply:
What does this mean? If you explain it to me, I can make sense of it, but the truth is I don't really understand it.

Reply:
"If your leader has a path like the one below " it is used to easily understand how it works

Reply:
Ok, now I add it again.